utam-compiler/src/main/java/utam/compiler/translator/ClassSerializer.java

```diff
@@ -95,6 +97,15 @@ public String toString() {
             .filter(method -> method.isPublic() || translationContext.getUsedPrivateMethods().contains(method.getDeclaration().getName()))
             .flatMap(method -> getMethodDeclaration(method).stream()).forEach(out::add);
     out.add(NEW_LINE);
+    source.getMethods().stream()
```

Comment:
i would separate lines 100-108 into private method with name addElementClassDeclaration
then it would be clear what it is

Reply:
Done in next commit.